app/models/browse_filter.rb

```diff
@@ -39,7 +39,10 @@ def contributions
   def options
     return {} unless context
 
-    { respond_path: ->(id) { context.respond_contribution_path(id)} }
+    {
+      respond_path: ->(id) { context.respond_contribution_path(id)},
+      view_path: ->(id) { context.contribution_path(id)}
```

Comment:
`view_path` probably only need to be rendered if the `peer_to_peer_matching` feature is turned on. IE we don't want [this conditional](https://github.com/rubyforgood/mutual-aid/blob/1882769d7dd6b7137531c962978e1857ba2465eb/app/javascript/pages/browse/TileListItem.vue#L40) to be true unless P2P is turned on

Reply:
Thanks for reminding that. I took care of that.